x-pack/legacy/plugins/monitoring/server/lib/metrics/__test__/__snapshots__/metrics.test.js.snap

```diff
@@ -461,7 +461,7 @@ Object {
     "label": "Requested",
     "metricAgg": "max",
     "timestampField": "beats_stats.timestamp",
-    "title": "Requests",
+    "title": "Requests Intake API",
```

Comment:
I liked your suggestion from the prev PR to change this to `Request Count Intake API` for consistency.

Reply:
Fixed. Thanks!